lib/EbookSource.php

```diff
@@ -1,14 +1,14 @@
 <?
-enum EbookSourceType{
-	case ProjectGutenberg;
-	case ProjectGutenbergAustralia;
-	case ProjectGutenbergCanada;
-	case InternetArchive;
-	case HathiTrust;
-	case Wikisource;
-	case GoogleBooks;
-	case FadedPage;
-	case Other;
+enum EbookSourceType: int{
```

Comment:
Yes, they need a type so that we can store them in the `EbookSources` table. I converted them to string/enum so that the table now looks like this:

```
MariaDB [se]> SELECT * FROM EbookSources;
+---------+-------------+------------------------------------------------------------------------+
| EbookId | Type        | Url                                                                    |
+---------+-------------+------------------------------------------------------------------------+
|     955 | pg          | https://www.gutenberg.org/ebooks/18640                                 |
|     955 | ia          | https://archive.org/details/phineasredux00trolrich                     |
|     956 | pg          | https://www.gutenberg.org/ebooks/72958                                 |
|     956 | ia          | https://archive.org/details/hunting-for-hidden-gold                    |
|     957 | pg          | https://www.gutenberg.org/ebooks/46674                                 |
|     957 | ia          | https://archive.org/details/stoveratyale00john                         |
|     958 | pg          | https://www.gutenberg.org/ebooks/72771                                 |
|     958 | hathi_trust | https://catalog.hathitrust.org/Record/101865194                        |
|     959 | wikisource  | https://en.wikisource.org/wiki/The_Kural_or_the_Maxims_of_Tiruvalluvar |
|     959 | ia          | https://archive.org/details/in.ernet.dli.2015.102804                   |
+---------+-------------+------------------------------------------------------------------------+
```